xunit-autolabeler-v2/ast_parser/core/analyze_test.py

```diff
@@ -0,0 +1,134 @@
+# Copyright 2020 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import unittest
+
+from ast_parser.core import analyze
+
+import mock
+
+import pytest
+
+
+TEST_DIR = os.path.join(
+    os.path.dirname(__file__),
+    'test_data/parser'
+)
+
+
+class AnalyzeBasicTests(unittest.TestCase):
+    @pytest.fixture(autouse=True)
+    def _get_analyze_result(self):
+        basic_test_dir = os.path.join(TEST_DIR, 'edge_cases')
+        analyze_result = analyze.analyze_json(
+            os.path.join(basic_test_dir, 'polyglot_snippet_data.json'),
+            basic_test_dir
+        )
+
+        self.grep_tags = analyze_result[0]
```

Comment:
What about:

self.grep_tags, self.source_tags, self.ignored_tags, self.source_methods = analyze_result

Reply:
Done. (This was too much to fit onto one line, so I wrapped the variiable list in parenthesis.)